src/workerd/server/workerd.capnp

```diff
@@ -466,7 +469,7 @@ struct Worker {
     }
   }
 
-  durableObjectUniqueKeyModifier @8 :Text;
+  durableObjectUniqueKeyModifier @9 :Text;
```

Comment:
In Cap'n Proto, the purpose of these `@N` numbers is to maintain binary backwards-compatibility of the serialization as new fields are added. If you change the numbers, you defeat the purpose -- this will break compatibility. Instead, leave the numbers the same on existing fields, and assign the next number in sequence to your new field. The numbers do not have to appear in order in the file -- in fact, that's why we have the numbers, so that you can still list fields in logical order rather than exactly the order in which they were added.